### ISSUE TITLE
exclude and order is no more respected

### ISSUE THREAD

**Issue by gsemet** [maxwroc/battery-state-card#624]
**Describe the bug**

With latest version, the exclude and order is no more respected:

**How to reproduce**
I want to show only 20% and bellow battery entities. The code worked before update to 1.3.1

**Expected behavior**
- all items above 20% battery shall not appear
- list should be ordered ascending

**YAML configuration**

```yaml
type: custom:battery-state-card
tap_action: more-info
secondary_info: '{last_changed}'
filter:
  include:
    - name: entity_id
      value: 'sensor.*battery_level*'
    - name: entity_id
      value: 'sensor.*_battery'
  exclude:
    - name: entity_id
      value: sensor.stibphone2_battery_level
    - name: state
      value: 20
      operator: '>'
sort: 
  - by: "state"
    desc: false
```

**Screenshots**
<img width="938" alt="image" src="https://github.com/maxwroc/battery-state-card/assets/133498/ba1ccf3b-594d-4abc-94bc-beecd2c419f9">

**Version**
3.1.0

**Comment by stijnb1234** [comment]
The filter `exclude` seems not to like the decimal numbers. When I set the representation to 100 instead of 100,0 the filter works fine.

**Comment by gsemet** [comment]
I tried `value: "20"` or `value: 20` or `value: 20.0`, none work

**Comment by maxwroc** [comment]
Fixed in v3.1.1
